src/scss/2022/components/_pie-chart.scss

```diff
@@ -0,0 +1,276 @@
+@use "../tools/fills";
+@use "../tools/color";
+@use "../tools/typography";
+@use "../tools/media-queries" as *;
+
+.cmp-pie-chart {
+  --object-1: 0;
+  --object-2: 0;
+  --object-3: 0;
+  --total: calc(var(--object-1) + var(--object-2) + var(--object-3));
+  
+  --turn-1: calc((var(--object-1) / var(--total)) * 360deg);
+  --turn-2: calc(((var(--object-2) / var(--total)) * 360deg) + var(--turn-1));
+  --turn-3: calc(((var(--object-3) / var(--total)) * 360deg) + var(--turn-2));
+
+  position: relative;
+
+  &__diagram {
+    position: relative;
+    width: 100%;
+    border-radius: 50%;
+    overflow: hidden;
+    border: 1px solid var(--color-white);
+  
+    &::after {
+      content: '';
+      display: block;
+      padding-top: 100%;
+    }
+  }
+
+  &__divider {
+    position: absolute;
+    top: 0;
+    left: calc(50% - 0.5px);
+    width: 1px;
+    height: 50%;
+    background-color: white;
+    z-index: 200;
+    transform-origin: 50% 100%;
+    
+    &--object-1 {
+      transform: rotate(var(--turn-1));
+    }
+    &--object-2 {
+      transform: rotate(var(--turn-2));
+    }
+    &--object-3 {
+      transform: rotate(var(--turn-3));
+    }
+  }
+
+  &__piece {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+
+    &--object-1 {
+      background-color: var(--color-black);
+      mask-image: conic-gradient(
+        #000 0deg,
+        #000 var(--turn-1),
+        transparent var(--turn-1)
+      );
+    }
+    
+    &--object-2 {
```

Comment:
On some magnifications (I see it at 100% on Chrome), object 2's striped background is causing a thin dashed line to appear horizontally midway down through the chart. Under key--item-1.
<img width="685" alt="Screen Shot 2022-06-14 at 10 57 19 AM" src="https://user-images.githubusercontent.com/25614178/173611546-1692c42e-2502-4145-b7de-31680ba1552a.png">

Reply:
I cannot replicate this on my computer. My guess is it’s not just a browser / zoom issue but also a screen resolution issue as well. I’ve tried out multiple configurations to replicate the issue. What I’m going to do is add a `will-change` property and that may fix it. So, let me know.

Reply:
I can still see it. I did wonder if a thin border or shadow on the mask might help. But I think that because the length of the line changes in addition to appearing and disappearing at different breakpoints, it is unlikely to cause confusion for the viewer as it might if it were more consistent. I am cool with letting it go if you are.